jbmc/unit/java_bytecode/java_bytecode_parser/InnerClasses.java

```diff
@@ -0,0 +1,26 @@
+public class InnerClasses {
```

Comment:
Could you add a non-public class to this file (in case that means there are two inner class attributes?) Unless that isn't true, in which case might not be worth it

Reply:
Added a non-public outer class.